### PR TITLE
Create dotnet.yml

### PR DESCRIPTION
**Pull request by mikiiier**
Auto run tests so i don't break anything

### CODE REVIEW

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
```
Comment (MovaUA) [inline .github/workflows/dotnet.yml:22]: Please the latest major version which is 7.

File: .github/workflows/dotnet.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - name: Restore dependencies
+      run: dotnet restore
```
Comment (MovaUA) [inline .github/workflows/dotnet.yml:24]: To have a reproducible build use the command with the following flags:
```bash
dotnet restore --lock-mode -warnaserror
```

File: .github/workflows/dotnet.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      working-directory: ./playground
+    - name: Build
+      run: dotnet build --no-restore
```
Comment (MovaUA) [inline .github/workflows/dotnet.yml:27]: be more strict to errors: add flag `-warnaserror`

**Comment by kiIIer** [comment]
closes #8

**Pull request closed without merging**